### ISSUE TITLE
Training command for ViT-L/14@336px

### ISSUE THREAD

**Issue by punisher1k** [YuYang0901/CLIP-spurious-finetune#2]
Hi @YuYang0901, thank you for sharing your great work.

I am trying to run your code, could you please update the training command to train the ViT-L/14@336px model on waterbird dataset and how you selected the checkpoint for evaluating

**Comment by punisher1k** [comment]
I also encountered some other errors:
- `python: can't open file 'tools/explain_predictions_imagenet.py': [Errno 2] No such file or directory`
- `detection_command_launchers.py: error: unrecognized arguments: --class_range 0 100`
Can you please update a runable and reproducible code, @YuYang0901?

**Comment by YuYang0901** [comment]
Hi @punisher1k,

Thanks for bringing these issues to my attention, and I'm glad to hear about your interest in our work!

I've just pushed a new commit that addresses the concerns you raised:

1. The `tools/explain_predictions_imagenet.py` file was missing from the repository and is now been added.

2. I've also corrected the command for `detection_command_launchers.py`. 

For training the ViT-L/14@336px model on the waterbird dataset, you can use the updated command as follows:

```
python run_expt.py --dataset waterbirds --model clip-vit --other_params <other necessary parameters>
```

Replace `<other necessary parameters>` with any additional flags or configurations required for your setup.

Regarding checkpoint selection for evaluation, as mentioned in captions of Table 4 and 5, "Both average and worst-group performance are evaluated with models early stopped at the highest worst-group accuracy on the validation set." 

Please try pulling the latest changes, and let me know if you encounter any further issues or have additional questions.

Best regards,
@YuYang0901

**Comment by punisher1k** [comment]
Thank you for your reply @YuYang0901, so what is your exact command to reproduce the reported result for ViT-L/14@336px in the paper? I meant your --other_params <other necessary parameters> part?

**Comment by YuYang0901** [comment]
Hi @punisher1k, I've updated the README file in the repository with the exact commands used for ViT-L/14@336px as reported in the paper. I hope this helps you in reproducing the results.
